recipes/pyrte_rrtmgp/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "pyrte_rrtmgp" %}
+{% set version = "0.0.6" %}
+{% set rte_rrtmgp_version = "1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/earth-system-radiation/pyRTE-RRTMGP/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: b161ac8b1ddca59eb994eb44ba8aabb9a596f50f24a8fd0886fea8c680e41a29
+  - url: https://github.com/earth-system-radiation/rte-rrtmgp/archive/refs/tags/v{{ rte_rrtmgp_version }}.tar.gz
+    fn: rte-rrtmgp-{{ rte_rrtmgp_version }}.tar.gz
+    sha256: e3830007529a3fb35727683ff990cac7e6792cdee4c9d9b07a55c969a97c95e5
+    folder: rte-rrtmgp
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - libuv ==1.44.2
+    - cmake ==3.26.4
+    - make
+
+  host:
+    - python
+    - pip
+    - scikit-build-core
+    - pybind11 >=2.10.0
+    - numpy >=1.21.0
```

This should be unpinned to use the globally pinned version.
```suggestion
    - numpy
```